app/views/replies/_reply.html.erb

```diff
@@ -42,9 +52,11 @@
 
       <% if user_signed_in? %>
         <% if reply.user.id == current_user.id %>
-          <%= link_to "Delete", {:action => 'destroy', :controller => "replies", :id => @reply.id, :new_thread_id => @new_thread.id}, :method => "delete", data: { confirm: 'Are you sure?' }, :class => 'reply-link edit-post'%>
+          <span id="edit-reply-btn-<%= reply.id %>" class="btn btn-default" reply-id="<%= reply.id %>"><i class="fa fa-pencil"></i></span>
+          <%= link_to "<i class='fa fa-trash'></i>".html_safe, {:action => 'destroy', :controller => "replies", :id => @reply.id, :new_thread_id => @new_thread.id}, :method => "delete", data: { confirm: 'Are you sure?' }, :class => 'reply-link edit-post btn btn-default'%>
         <% end %>
       <%end%>
+      <%= link_to "<i class='fa fa-share-alt'></i>".html_safe, new_thread_path + "#reply_" + reply.id.to_s, class: 'btn btn-default' %>
```

Comment:
Well I don't know as well. Here is [the line](https://github.com/sonalkr132/cat-forum/blob/github_master/app/views/replies/_reply.html.erb#L2) that assigns it. Might be some legacy code. Let me check it. If everything is alright we can replace all instances of `@reply` with `reply`.